src/DotNetWorker.Grpc/Http/GrpcHttpRequestData.cs

```diff
@@ -146,13 +155,18 @@ public void Dispose()
 
         private IReadOnlyCollection<IHttpCookie> ToHttpCookies(string cookieString)
         {
-            var separateCookies = cookieString.Split(";");
+            var separateCookies = cookieString.Split(';');
 
             List<IHttpCookie> httpCookiesList = new List<IHttpCookie>(separateCookies.Length);
 
             for (int c = 0; c < separateCookies.Length; c++)
             {
-                var splitArray = separateCookies[c].Split("=", StringSplitOptions.RemoveEmptyEntries);
+#if NET5_0_OR_GREATER
+                var separator = '=';
+#else
+                var separator = new[] { '=' };
```

Comment:
Unrelated to this PR. But the separator doesn't need to be allocated for each iteration of the loop. Could be moved out.